include/string.h

```diff
@@ -49,8 +49,7 @@ int memcmp (const void * : itype(_Array_ptr<const void>) byte_count(n),
 void *memchr (const void *src : itype(_Array_ptr<const void>) byte_count(n),
               int c,
               size_t n)
-  : itype(_Array_ptr<void>) byte_count(n);
-
+  : bounds(src, (_Array_ptr<char>) src + n);
 char *strcpy (char *__restrict, const char *__restrict);
 char *strncpy (char *__restrict, const char *__restrict, size_t);
```

Comment:
Can we convert `strncpy`?

Reply:
I am converting strn* and strl* functions. I will push them in a different commit whenever I am done with all of them.